File: swagger/models/get_gift_card_balance_response.py
from .base_model import BaseModel


class GetGiftCardBalanceResponse(BaseModel):
	"""
	no description available

	Attributes:

		barcode_id
		The gift card's barcode ID.

		remaining_balance
		The gift card's remaining balance.
	"""

	input_map = {
		'BarcodeId': 'barcode_id',
		'RemainingBalance': 'remaining_balance',
		}

	output_map = {
		'barcode_id': 'BarcodeId',
		'remaining_balance': 'RemainingBalance',
		}

	barcode_id = None
	remaining_balance = None

File: swagger/Models/GetGiftCardBalanceResponse.php
<?php

namespace App\MindBody\Models;

/**
 * no description available
 */
class GetGiftCardBalanceResponse extends BaseModel
{
	/**
	 * The gift card's barcode ID.
	 * 
	 * @var string 
	 */
	public $barcodeId;
	/**
	 * The gift card's remaining balance.
	 * 
	 * @var number format:double
	 */
	public $remainingBalance;

	protected function getInputMap()
	{
		return [
			'BarcodeId' => ['barcodeId', 'string', null],
			'RemainingBalance' => ['remainingBalance', 'number', 'double'],
			];
	}

	protected function getOutputMap()
	{
		return [
			'barcodeId' => ['BarcodeId', 'string', null],
			'remainingBalance' => ['RemainingBalance', 'number', 'double'],
			];
	}
}
